sharktank/tests/models/flux/flux_test.py

```diff
@@ -25,6 +26,7 @@
     make_random_theta,
 )
 from sharktank.models.flux.flux import FluxModelV1, FluxParams
+from sharktank.models.flux.compile import iree_compile_flags
```

Comment:
Nit: Unused import

Reply:
This is still used in some tests. Potentially we can refactor the tests to use the config presets. I will leave this for another PR.